README.md

```diff
@@ -28,6 +28,15 @@
 
 # Getting started
 
+### Script 
+1. Download this repository extract and open the folder
+2. Run `newproject.sh` script to create a copy with the name you want 
+3. On `app`, change the package on the `mode` lazy var
+4. On `theme.xml` & `colors.xml` set your application primary and secondary colors 
+
+And you're ready to start working on your new app.
+
+### Manual
 1. Download this repository extract and open the template folder on Android Studio
 2. Rename the app package `io.bloco.template`
 3. Check if the manifest package was renamed along with the package
```

Comment:
The step 4 is no longer needed.

Reply:
We either add step 6 to the Script as well, or take it out.

Reply:
`Make sure the method `checkTestMode` inside `AndroidApplication` includes a test class that exists` no longer applies